api/inventorypb/agents.proto

```diff
@@ -325,6 +325,28 @@ message RDSExporter {
   bool enhanced_metrics_disabled = 10;
 }
 
+// ExternalExporter runs on any Node.
```

Comment:
Not on any – it can't run on remote nodes. So – Generic or Container.

Reply:
Actually it can, based on next task related to external exporters it will be able to work on remote task too.

Reply:
What is the definition of remote node?

Reply:
Some node to which we don't have access.